src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -625,6 +631,7 @@ void ColorControlServer::startColorLoop(EndpointId endpoint, uint8_t startFromSt
 
     colorHueTransitionState->stepsRemaining = static_cast<uint16_t>(time * TRANSITION_TIME_1S);
     colorHueTransitionState->stepsTotal     = static_cast<uint16_t>(time * TRANSITION_TIME_1S);
+    colorHueTransitionState->timeRemaining  = MAX_INT16U_VALUE;
```

Comment:
is MAX_INT16U_VALUE guaranteed to be defined?

Reply:
I assume so as it's already being used in the file. It's defined in src/app/util/af-types.h.